lib/auth/init.go

```diff
@@ -680,11 +681,11 @@ func initCluster(ctx context.Context, cfg InitConfig, asrv *Server) error {
 		}
 		span.AddEvent("completed creating database object import rules")
 
-		span.AddEvent("creating preset health check config")
-		if err := createPresetHealthCheckConfig(ctx, asrv); err != nil {
+		span.AddEvent("creating preset health check configs")
+		if err := createPresetHealthCheckConfigs(ctx, asrv); err != nil {
```

Comment:
Should we make it a migration to ensure we create these even for older clusters instead of adding this feature only for newly bootstrapped clusters?